robyn/env_populator.py

```diff
@@ -1,21 +1,20 @@
+from distutils.command.config import config
 import os 
 import logging
 from pathlib import Path
 
 
-
-# Path to the root of the project
-ROOT_DIR = Path(__file__).parent.parent
-
-# Path to the environment variables
-CONFIG_PATH = ROOT_DIR / 'robyn.env'
-
 #set the logger that will log the environment variables imported from robyn.env and the ones already set
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
+
 # parse the configuration file returning a list of tuples (key, value) containing the environment variables
-def parser(config_path=CONFIG_PATH):
+def parser(config_path=None, project_root = None):
+    """Find robyn.env file in root of the project"""
+    if config_path is None:
+        config_path = Path(project_root) / "robyn.env"
+
     """Parse the configuration file"""
```

Comment:
@Shending-Help , this should be the docstring of the function as this is the main purpose.

You can make the comment below  as a `#`comment.
> """Find robyn.env file in root of the project"""

Docstrings are reflected in the function metadata and it doesn't make a lot of sense to add two of them.

Reply:
@sansyrox  fixed it